src/test/rpc/Simulate_test.cpp

```diff
@@ -662,7 +720,7 @@ class Simulate_test : public beast::unit_test::suite
                             BEAST_EXPECT(finalFields[sfDomain] == newDomain);
                         }
                     }
-                    BEAST_EXPECT(metadata[sfTransactionIndex.jsonName] == 1);
+                    BEAST_EXPECT(metadata[sfTransactionIndex.jsonName] == 0);
```

Comment:
Explain this change please

Reply:
I forgot to close the ledger after submitting the `SignerListSet` transaction above, so this transaction was the second in that particular ledger. I'm closing the ledger now (to ensure that no transactions are accidentally queued), so the index changes as a result.